s3_management/manage.py

```diff
@@ -61,6 +62,7 @@ def nightly_packages_to_show(self: S3IndexType) -> Set[str]:
         # sorting, sorts in reverse to put the most recent versions first
         all_sorted_packages = sorted(
             {self.normalize_package_version(obj) for obj in self.objects},
+            key=lambda v: parse(v.split('-')[1]),
```

Comment:
Wouldn't parse(v) work as well/with more precision?

Reply:
Oh sorry, never mind. I didn't realize v was not just the version, but the entire package name!

Reply:
Yeah the naming is a bit confusing here. I have renamed the lambda variable.